assets/sass/components/_chart-colours.scss

```diff
@@ -0,0 +1,41 @@
+.app-chart-colour--a {
+  background: govuk-tint(govuk-colour("blue"), 51%); // miro ~ 90aadc
+}
+
+.app-chart-colour--b {
+  background: govuk-tint(govuk-colour("dark-blue"), 18%); // miro ~ 2f5598
+}
```

Comment:
Would be useful to have these 2 colours in variables like the `$heat-map-colour` one?

Also, maybe we'll use these for other meanings but it seems like at the moment these means "positive behaviours/case notes" and "negative behaviours/case notes". 
So maybe we could call them `-pos`/`-neg` or even `-positive`/`-negative`, just an idea.

Reply:
i didn't go for pos/neg because there will be other charts in future and they may use a larger palette. same with the classes: i'd rather they could be used in any chart rather than be tied to a word that doesn't then make sense.

as for using variables, that's a good idea but the colours themselves are very WIP at the moment and justin is looking into it. i basically tried to tint GDS colours to get as near to miro designs as possible.

i'm gonna leave their naming for now with the view to it all possibly changing anyway

Reply:
I don't think it matters much at this stage as you said but I think I prefer the words even if we get a larger palette as they express the meaning of the colour rather than the...colour (e.g. "warning" > "yellow").

If a colour would be used in different contexts nothing wrong with having more than one variables with the same value if they refer to different meanings.

But yeah, don't worry about the names, it was a thought ☺️